Dockerfile

```diff
@@ -1,12 +1,12 @@
-# syntax=docker/dockerfile:1
-
-FROM golang:1.17
-
+FROM golang:1.22.5 AS builder
```

Comment:
Let's leave the go version the same as before, we should use the same version for development and deployment.

Reply:
It looks like the repo's version of the dockerfile has been updated, please modify this to line up with the version in go.mod.